packages/eslint-config/package.json

```diff
@@ -26,24 +26,22 @@
     "eslint-config-airbnb-base": "^15.0.0",
     "eslint-config-prettier": "^8.9.0",
     "eslint-import-resolver-typescript": "^3.5.5",
-    "eslint-plugin-chai-expect": "^3.0.0",
-    "eslint-plugin-chai-friendly": "^0.7.2",
     "eslint-plugin-eslint-comments": "^3.2.0",
     "eslint-plugin-import": "^2.28.0",
     "eslint-plugin-jest": "^27.2.3",
     "eslint-plugin-jest-dom": "^5.0.1",
     "eslint-plugin-jest-formatting": "^3.0.0",
-    "eslint-plugin-jsdoc": "^46.4.5",
     "eslint-plugin-jsx-a11y": "^6.7.1",
-    "eslint-plugin-mocha": "10.1.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^5.0.0",
     "eslint-plugin-react": "^7.33.1",
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-testing-library": "^5.11.0",
     "eslint-plugin-typescript-sort-keys": "^2.3.0",
     "eslint-plugin-unicorn": "^48.0.1",
-    "eslint-plugin-you-dont-need-lodash-underscore": "^6.12.0"
+    "eslint-plugin-vitest": "^0.2.8",
+    "eslint-plugin-you-dont-need-lodash-underscore": "^6.12.0",
+    "lodash.merge": "^4.6.2"
```

Comment:
I don't love bringing lodash into the mix as a main dep, any way we could avoid that?

Reply:
There's not really any good to merge two deeply nested objects in vanilla JS without it.

Reply:
It's at least only bringing in the `merge` method and not all of Lodash.